### DISCUSSION TITLE
Have you considered adding a rerank model after completing the similarity search to further filter the results?

### DISCUSSION THREAD

**Discussion by ghost** [vanna-ai/vanna#774] [Q&A]

**Comment by ghost** [comment]
In real-world RAG product production environments, rerank models play a significantly noticeable role.

**Comment by wangshili123** [comment]
And I found that the search results were not very friendly, and the same problem was not retrieved

**Comment by ghost** [comment]
ddl可以增加description。rerank更多是语义上的。embedding的模型，有些对中文支持不是特别好。
